testing/regress/ecl-test

```diff
@@ -472,13 +480,27 @@ class RegressMain:
 
 
             if self.args.func == 'list':
-                self.listClusters()
+                self.listClusters()   
```

Comment:
Please be careful and do not insert unnecessary spaces. Check the other line as well.

Reply:
Noted